File: Conceitos de LPs/Tipos de Dados/Matrizes Associativas/associativas.py
#Python
#Suporte Nativo

# Definição e inicialização
matriz = {
    "um": 1,
    "dois": 2,
    "tres": 3
}

# Acessar elementos
print("Valor para 'dois':", matriz["dois"])

# Remover um elemento
del matriz["tres"]

# Iterar pela matriz
for chave, valor in matriz.items():
    print(f"{chave}: {valor}")

File: Conceitos de LPs/Tipos de Dados/Matrizes Associativas/associativas.php
<?php
//PHP
//Suporte Nativo

// Definição e inicialização
$matriz = array(
    "um" => 1,
    "dois" => 2,
    "tres" => 3
);

// Acessar elementos
echo "Valor para 'dois': " . $matriz["dois"] . "\n";

// Remover um elemento
unset($matriz["tres"]);

// Iterar pela matriz
foreach ($matriz as $chave => $valor) {
    echo "$chave: $valor\n";
}
?>
